### ISSUE TITLE
Reported "Connection" header is always "close"

### ISSUE THREAD

**Issue by lukas-m** [kennethreitz/httpbin#432]
When Connection:keep-alive is used, the JSON response body always contains Connection:close.

See communication log (copied from Wireshark):

GET /get HTTP/1.1
Connection: keep-alive
Host: httpbin.org
Accept-Encoding: gzip, deflate

HTTP/1.1 200 OK
Connection: keep-alive
Server: meinheld/0.6.1
Date: Wed, 07 Mar 2018 15:58:22 GMT
Content-Type: application/json
Access-Control-Allow-Origin: *
Access-Control-Allow-Credentials: true
X-Powered-By: Flask
X-Processed-Time: 0
Content-Length: 197
Via: 1.1 vegur

{
  "args": {}, 
  "headers": {
    "Accept-Encoding": "gzip, deflate", 
    "Connection": "close", 
    "Host": "httpbin.org"
  }, 
  "origin": "46.13.8.130", 
  "url": "http://httpbin.org/get"
}

**Comment by sigmavirus24** [comment]
Please review open and closed issues before opening a new one. This has been discussed in the past. There's a proxy between the service and you. That proxy makes requests to the service with `Connection: close` as it is allowed to by the specification. There's nothing we can do short of moving it to custom infrastructure which this team doesn't have time to maintain or operate.

**Comment by lukas-m** [comment]
I am sorry for opening new issue. I have found the duplicate at https://github.com/kennethreitz/httpbin/issues/377.

I was surprised that even the header in server (proxy) reply was `Connection:keep-alive`, so I have played with it locally. I can confirm that (if requesting keep-alive connection):

-  When httpbin is used with nginx (proxy), JSON body reports `Connection:close`. The response contains `Connection:keep-alive` header and the connection is persistent.

-  When httpbin is running on plain Unix (no proxy), JSON body reports `Connection:keep-alive`. The response contains `Connection:close` header and the connection is not persistent.

Thanks for your hard work.
